### DISCUSSION TITLE
MS store claims this has in-app purchases. What's that about?

### DISCUSSION THREAD

**Discussion by Githubber616** [File-New-Project/EarTrumpet#1132] [Q&A] [answered]
MS store claims this has in-app purchases. What's that about?
Is that MS being stupid?

**Comment by riverar** [comment, marked as answer]
@Githubber616 Yep. Years ago, we were trying out "DLC" packages as a potential way to deliver addons. One of the side effects of this was that the Store would think we sold them. Fast forward to 2022, they still haven't fixed this, despite my repeated requests/attempts.
